.github/actions/javascript/markPullRequestsAsDeployed/action.yml

```
@@ -28,5 +28,5 @@ inputs:
         description: "Web job result ('success', 'failure', 'cancelled', or 'skipped')"
         required: true
 runs:
-    using: "node16"
+    using: "node20"
```

NAB but would be nice to standardize on `'` or `"`

```suggestion
    using: 'node20'
```